vpc.go

```diff
@@ -25,6 +25,11 @@ type VPCService interface {
 	Update(ctx context.Context, vpcID string, uvpl *UpdateVPCPayload) (*VPC, error)
 	Create(ctx context.Context, cvpl *CreateVPCPayload) (*VPC, error)
 	Delete(ctx context.Context, vpcID string) error
+	CreateVPCNetworkInterface(ctx context.Context, networkID string, cnip *CreateNetworkInterfacePayload) (*NetworkInterface, error)
+	UpdateVPCNetworkInterface(ctx context.Context, networkID string, netInterfaceID string, unip *UpdateNetworkInterfacePayload) (*NetworkInterface, error)
+	DeleteVPCNetworkInterface(ctx context.Context, networkID string, netInterfaceID string) error
+	GetVPCNetworkInterface(ctx context.Context, networkID string, netInterfaceID string) (*NetworkInterface, error)
+	ListNetworkInterface(ctx context.Context) ([]*NetworkInterface, error)
```

Comment:
with list, I think you can pass search options for this. such as list network interface by VPC Network ID or list all network interface